nav2_behavior_tree/include/nav2_behavior_tree/bt_cancel_action_node.hpp

```diff
@@ -89,7 +91,7 @@ class BtCancelActionNode : public BT::ActionNodeBase
 
     // Make sure the server is actually there before continuing
     RCLCPP_DEBUG(node_->get_logger(), "Waiting for \"%s\" action server", action_name.c_str());
-    if (!action_client_->wait_for_action_server(1s)) {
+    if (!action_client_->wait_for_action_server(wait_for_service_timeout_)) {
       RCLCPP_ERROR(
         node_->get_logger(), "\"%s\" action server not available after waiting for 1 s",
```

Comment:
Not 1s

Reply:
You are totally right, this is wrong in the main branch as well. Should I create a PR targeting the main branch fixing this, which will be backported later? Or do you want me to include it here as well?
https://github.com/ros-navigation/navigation2/blob/main/nav2_behavior_tree/include/nav2_behavior_tree/bt_cancel_action_node.hpp#L96